src/index.ts

```diff
@@ -0,0 +1,35 @@
+import type { FastifyInstance } from "fastify";
+import "dotenv/config";
+import { closeDB } from "./db.js";
+import { buildApp } from "./app.js";
+
+let server: FastifyInstance;
+const PORT: number = Number(process.env.PORT) || 3000;
+const HOST: string = "0.0.0.0";
+
+const start = async () => {
+  server = await buildApp();
+  server.listen(
+    { port: PORT, host: HOST },
+    (err: Error | null, _address: string) => {
+      if (err) {
+        server.log.error(err);
+        process.exitCode = 1;
+      }
+    }
+  );
+};
+
+const shutdown = async (signal: string) => {
+  server.log.info(`${signal} received, shutting down...`);
```

Comment:
pino is a json logger meaning that you want to pass json object to if, example, that way when you monitor your app it's easier to understnad what happening instead of reading a really long string
```suggestion
  server.log.info({signal}, 'Shutting down...');
```